client/app/components/PermissionsEditorDialog/index.jsx

```diff
@@ -16,6 +16,7 @@ import notification from "@/services/notification";
 import User from "@/services/user";
 
 import "./index.less";
+import Link from "../Link";
```

Comment:
Please use full path here (`@/components/Link`) and move this import upper (to other @-imports)